src/shared_modules/content_manager/doc/components/OFFLINE_DOWNLOADER.md

```diff
@@ -0,0 +1,26 @@
+# Offline Downloader stage
+
+## Details
+
+The `offline downloader` stage is part of the Content Manager orchestration and is in charge of copying an input file from the local filesystem to be processed by the following stages. The input file hash is calculated and stored in order to avoid processing the same file multiple times in the next orchestration executions.
+
+The copy can be made into any of the two following output directories:
+- Downloads folder: If the input file is compressed.
+- Content folder: If the input file is not compressed.
+
+If the copy is successful, this stage also updates the context [data paths](../../src/components/updaterContext.hpp) with the copy destination path.
+
+> Note: Despite the class name, there is no such download performed. The copy is made entirely locally.
+
+## Relation with the UpdaterContext
+
+The context fields related to this stage are:
+
+- `configData`
+  + `url`: Used as input file path.
```

Comment:
We should mention the format that this URL should have: "file://"

Reply:
Added a note. It's not mandatory though.